aws/resource_aws_appsync_datasource_test.go

```diff
@@ -46,13 +46,14 @@ func TestAccAwsAppsyncDatasource_es(t *testing.T) {
 }
 
 func TestAccAwsAppsyncDatasource_lambda(t *testing.T) {
+	Desc := "appsync datasource"
```

Comment:
FYI, starting variable names with a capital letter is special in Go (makes them exportable). While it shouldn't matter in this case, its generally best practice to lowerCamelCase unless otherwise necessary.